sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -367,6 +367,8 @@ case class OuterReference(e: NamedExpression)
   override def exprId: ExprId = e.exprId
   override def toAttribute: Attribute = e.toAttribute
   override def newInstance(): NamedExpression = OuterReference(e.newInstance())
+  override def sql: String = e.sql
+  override def toString: String = e.toString
```

Comment:
Any reason to override this? If we keep it unchanged, will it be easier for us to find whether it is an outer reference in the plans?